cbqueryx/query_options_test.go

```diff
@@ -2,17 +2,64 @@ package cbqueryx
 
 import (
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestEncodeQueryOptions(t *testing.T) {
-	opts := &Options{
-		Statement: "SELECT *",
-	}
+	t.Run("statement", func(t *testing.T) {
+		opts := &Options{
+			Statement: "SELECT *",
+		}
 
-	optsJson, err := opts.encodeToJson()
-	assert.NoError(t, err)
+		optsJson, err := opts.encodeToJson()
+		assert.NoError(t, err)
 
-	assert.Equal(t, `{"statement":"SELECT *"}`, string(optsJson))
+		assert.Equal(t, `{"statement":"SELECT *"}`, string(optsJson))
+	})
+
+	t.Run("scan_wait", func(t *testing.T) {
+		opts := &Options{
+			ScanWait: 2500 * time.Millisecond,
+		}
+
+		optsJson, err := opts.encodeToJson()
+		assert.NoError(t, err)
+
+		assert.Equal(t, `{"scan_wait":"2.5s"}`, string(optsJson))
```

Comment:
I have moderate concern that the actual value here is at the whim of the Go version being used, but we can cross that bridge if the issue ever arises.